core/performance_test/peakflops/03_Cabana_peakflops.cpp

```diff
@@ -62,7 +55,9 @@ struct data_t
 // Helper functions.
 //---------------------------------------------------------------------------//
 // Move function using the array-of-struct-of-arrays synatx.
-void movePx(data_t *__restrict__ a,  data_t *__restrict__ x0,
+
+void //__attribute__ ((noinline))
```

Comment:
Is the attribute needed here?

Reply:
This is a tricky one. GCC generates better assembly with the attribute, but in an ideal world you'd never tell it not to inline... 

(gcc bug?)

Reply:
hmmmm - so I see it is commented out which is why I originally was wondering if we needed it. I would say either remove if not needed. If needed, but not portable, wrap in some type of macro.

Reply:
@rfbird I guess the simplest thing to do is to remove it for now.

Reply:
Is this going to be removed or uncommented?

Reply:
@guangyechen @rfbird lets make a call here and resolve this one

Reply:
I mad the call to remove the comment attribute, and leave the `// NOTE` above